### ISSUE TITLE
404 Href in Javadoc on References class

### ISSUE THREAD

**Issue by SevaSafris** [opentracing/opentracing-java#303]
The Javadoc on the References class has 2 links that not found.

**Comment by austinlparker** [comment]
Nice catch, thanks. We're planning to launch an updated version of the site very soon so once that's out I'll update these links.

**Comment by SevaSafris** [comment]
@austinlparker, if I'm not mistaken, I think those href(s) should point to: https://github.com/opentracing/specification/blob/master/specification.md#references-between-spans

**Comment by austinlparker** [comment]
Yes, but one part of the new site is that the specification will be available at opentracing.io itself, so the link should point there when it's available. You can see it at https://opentracing.netlify.com/specification/ if you'd like.
